src/field.rs

```diff
@@ -123,20 +183,37 @@ macro_rules! make_field {
         pub struct $elem(u128);
 
         impl $elem {
-            fn try_from_bytes(bytes: &[u8], mask: u128) -> Result<Self, FieldError> {
+            /// Attempts to instantiate an `$elem` from the first `Self::BYTES` little-endian bytes
```

Comment:
I wonder if we should use big-endian instead, because we may end up using this routine and `Vec<u8>::from::<FieldElement>()` to handle encoded values on the wire, and big endian (network order) may be more natural there.